### DISCUSSION TITLE
[3.6.0] Infinite Loading?

### DISCUSSION THREAD

**Discussion by ElaineLMN** [Raytwo/ARCropolis#394] [Issues] [answered]
After updating, the game is unable to enter any kind of match. Is there a fix for this?

**Comment by arcenedon** [comment]
> After updating, the game is unable to enter any kind of match. Is there a fix for this?

Having the same issue as well.

**Comment by CupofJoe2** [comment]
> After updating, the game is unable to enter any kind of match. Is there a fix for this?

Same here

**Reply by MewtwoFanz** [reply]
> After updating, the game is unable to enter any kind of match. Is there a fix for this?

I am also having this issue.

**Comment by EvanScribe** [comment]
Same issue here, not sure what it is! At least I know it wasn't just me

**Comment by maxens-wlfr** [comment, marked as answer]
Yes, I seem to be affected as well

Edit : the update fixed it !
